spec/services/booking_responder/booking_request_processor_spec.rb

```diff
@@ -13,17 +13,53 @@
 
   subject { described_class.new(staff_response) }
 
-  it '#process_request' do
-    expect {
-      subject.process_request message do
-        visit.accept!
+  describe '#process_request' do
+    context 'happy path' do
```

Comment:
happy path? Surely is a better way to describe the process. I have not idea what the "happy path" entails, same for the "sad path"